docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md

```diff
@@ -46,41 +46,41 @@ For details, see the [TracePipeline specification file](https://github.com/kyma-
 <!-- TracePipeline v1alpha1 telemetry.kyma-project.io -->
 | Parameter         | Description                                   |
 | ---------------------------------------- | ---------|
-| **spec.output** | Configures the trace receiver of a TracePipeline. |
-| **spec.output.otlp** | Defines an output using the OpenTelmetry protocol. |
+| **spec.output** | Defines a destination for shipping trace data. Only one can be defined per pipeline. |
+| **spec.output.otlp** | Configures the underlying Otel Collector with an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/otlpexporter/README.md). If you switch `protocol`to `http`, an [OTLP HTTP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter) is used. |
 | **spec.output.otlp.authentication** | Defines authentication options for the OTLP output |
 | **spec.output.otlp.authentication.basic** | Contains credentials for HTTP basic auth |
-| **spec.output.otlp.authentication.basic.password** | Contains the basic auth password or a secret reference |
+| **spec.output.otlp.authentication.basic.password** | Contains the basic auth password or a secret reference. |
 | **spec.output.otlp.authentication.basic.password.value** |  |
 | **spec.output.otlp.authentication.basic.password.valueFrom** |  |
-| **spec.output.otlp.authentication.basic.password.valueFrom.secretKeyRef** |  |
+| **spec.output.otlp.authentication.basic.password.valueFrom.secretKeyRef** | Reference to a key in a Secret. You must provide `name` and `namespace` of the Secret, as well as the name of the `key`. |
 | **spec.output.otlp.authentication.basic.password.valueFrom.secretKeyRef.key** |  |
 | **spec.output.otlp.authentication.basic.password.valueFrom.secretKeyRef.name** |  |
 | **spec.output.otlp.authentication.basic.password.valueFrom.secretKeyRef.namespace** |  |
-| **spec.output.otlp.authentication.basic.user** | Contains the basic auth username or a secret reference |
+| **spec.output.otlp.authentication.basic.user** | Contains the basic auth username or a secret reference. |
 | **spec.output.otlp.authentication.basic.user.value** |  |
 | **spec.output.otlp.authentication.basic.user.valueFrom** |  |
-| **spec.output.otlp.authentication.basic.user.valueFrom.secretKeyRef** |  |
+| **spec.output.otlp.authentication.basic.user.valueFrom.secretKeyRef** | Reference to a key in a Secret. You must provide `name` and `namespace` of the Secret, as well as the name of the `key`. |
 | **spec.output.otlp.authentication.basic.user.valueFrom.secretKeyRef.key** |  |
 | **spec.output.otlp.authentication.basic.user.valueFrom.secretKeyRef.name** |  |
 | **spec.output.otlp.authentication.basic.user.valueFrom.secretKeyRef.namespace** |  |
 | **spec.output.otlp.endpoint** | Defines the host and port (<host>:<port>) of an OTLP endpoint. |
 | **spec.output.otlp.endpoint.value** |  |
```

Comment:
Here documentation is missing which existed in the docu before: https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline/